visualization/app/codeCharta/ui/customConfigs/customConfigs.component.ts

```diff
@@ -74,8 +80,42 @@ export class CustomConfigsController implements FilesSelectionSubscriber {
 		CustomConfigHelper.applyCustomConfig(configId, this.storeService, this.threeCameraService, this.threeOrbitControlsService)
 	}
 
-	removeCustomConfig(configId) {
+	removeCustomConfig(configId, customConfigItemIndex, dropDownCustomConfigItemGroupIndex) {
+		const { dropDownCustomConfigItemGroups } = this._viewModel
+
 		CustomConfigHelper.deleteCustomConfig(configId)
+		dropDownCustomConfigItemGroups[dropDownCustomConfigItemGroupIndex].customConfigItems.splice(customConfigItemIndex, 1)
+
+		if (dropDownCustomConfigItemGroups[dropDownCustomConfigItemGroupIndex].customConfigItems.length === 0) {
+			dropDownCustomConfigItemGroups.splice(dropDownCustomConfigItemGroupIndex, 1)
+			this._viewModel.visibleEntries--
+			this.previousEntries--
+
+			// eslint-disable-next-line unicorn/consistent-destructuring
+			if (this._viewModel.visibleEntries === 0) {
+				this._viewModel.showNonApplicableButton = false
+			}
+		}
+	}
+
+	initCurrentPagination() {
+		let applicableEntries = 0
+
+		while (this._viewModel.dropDownCustomConfigItemGroups[applicableEntries]?.hasApplicableItems) {
+			applicableEntries++
+		}
+
+		this._viewModel.showNonApplicableButton = applicableEntries < this._viewModel.dropDownCustomConfigItemGroups.length
+		this._viewModel.visibleEntries = applicableEntries
+	}
+
+	toggleNonApplicableItems() {
+		if (this._viewModel.visibleEntries < this._viewModel.dropDownCustomConfigItemGroups.length) {
```

Comment:
We might want to use a function to provide a nicer name for this condition that could be used here and in the template. I could not think of a great name right away. Maybe something along the lines of `isExpanded()`.